### PR TITLE
DOCSP-45807: v3.1 release

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-45807>

### Staging Links

<li><a href=https://deploy-preview-347--mongodb-docs-csharp.netlify.app/whats-new>whats-new</a></li>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -45,6 +45,27 @@ The 3.1 driver release includes the following new features:
   namespace. The driver can now serialize ``ImmutableArray`` objects, and serialization
   for other immutable collections is more memory efficient.
 
+- Adds support for the token field time and array field expressions with Atlas Search
```
Comment (jordan-smith721) [inline source/whats-new.txt:48]:
```suggestion
- Adds support for the token field type and array field expressions with Atlas Search
```

File: source/whats-new.txt [hunk 2]
```diff
@@ -45,6 +45,27 @@ The 3.1 driver release includes the following new features:
...
+  builders for the ``equals`` operator. To learn more about using Atlas Search with the
+  {+driver-short+}, see :ref:`csharp-atlas-search`.
+
+- Adds support for sequential pagination in Atlas Search.
+
+- Adds support for valid SRV hostnames with fewer than 3 parts.
+
+- Adds support for the ``Exists``, ``IsMissing``, and ``IsNullOrMissing`` methods
+  in MQL queries.
```
Comment (jordan-smith721) [inline source/whats-new.txt:57]: We don't externally refer to "MQL" anymore:
```suggestion
- Adds support for the ``Exists``, ``IsMissing``, and ``IsNullOrMissing`` methods
  in MongoDB Query API filters.
```

**Review by jordan-smith721 — APPROVED** [review]
A couple quick fixes, otherwise LGTM

File: source/whats-new.txt (hunk 1 above)
Comment (mcmorisi) [reply source/whats-new.txt:48]: 🤦 good catch

**Pull request merged**